utils/nlink/src/wlan_nlink_srv.c

```diff
@@ -345,7 +345,7 @@ int nl_srv_unregister(tWlanNlModTypes msg_type, nl_srv_msg_callback msg_handler)
 	return 0;
 }
 
-void *nl80211hdr_put(struct sk_buff *skb, uint32_t portid,
+static inline void *nl80211hdr_put(struct sk_buff *skb, uint32_t portid,
 		     uint32_t seq, int flags, uint8_t cmd)
 {
 	struct genl_family *cld80211_fam = cld80211_get_genl_family();
```

Comment:
"Fix compilation" wouldn't this cause problems when the clashing implementations for `nl80211hdr_put` are different?

Reply:
Perhaps this function should have just been `static` and removed from the header, if it's only used within `wlan_nlink_srv.c`? Or is it used externally too?

Reply:
I guess it has its roots somewhere around here
https://github.com/sonyxperiadev/kernel/blob/aosp/LA.UM.9.12.r1/net/wireless/nl80211.c#L690
I will push the proper fix